plugins/woocommerce-blocks/bin/webpack-configs.js

```diff
@@ -327,7 +331,8 @@ const getFrontConfig = ( options = {} ) => {
 			// @see https://github.com/Automattic/jetpack/pull/20926
 			chunkFilename: `[name]-frontend${ fileSuffix }.js?ver=[contenthash]`,
 			filename: `[name]-frontend${ fileSuffix }.js`,
-			uniqueName: 'webpackWcBlocksJsonp',
+			uniqueName: 'webpackWcBlocksFrontendJsonp',
```

Comment:
When I initially started optimizing the build i got runtime errors because we mistakenly had the same unique name for different bundles. This is a no-no. If 2 bundles on the same page have same unique name they can accidentally load packages from each others bundles and this was happening to me. The front end bundle would try load from the main vendor bundle rather than the frontend vendor bundle, causing issues if some parts of a dependency were tree shaken in that vendor bundle but not in mine. I think for consistency each entry *needs* a unique name to avoid this kind of conflict in future.